classes/01class/exercises/c01-git03/drii-cavalcanti/c01-git03.txt

```diff
@@ -0,0 +1,13 @@
+1. Did you notice any difference in the size of the repo before and after adding the big file?
+   Yes, the size improves as commits are asked. It means that even if we remove the files from the project with a commit, the files still exist in the repository history.
+
+2. What is the reason for this problem?
+   It happens because git keeps track of each and every line change we made.
+
+3. How do you remove something from Git history? Which commands would you use? What are the consequences?
```

Comment:
Here you would need to drop/remove the commits from the remote repository.
It can be done by using `git rebase -i HEAD~X` , then removing the commmit line or filling with `drop`. 
After that, you need to force push the changes.

Just `git gc` we'll not be enough.

Can you update this, please?